net/pfSense-pkg-Avahi/files/usr/local/share/pfSense-pkg-Avahi/info.xml

```diff
@@ -7,5 +7,10 @@
 		<descr><![CDATA[Avahi is a system which facilitates host and service discovery in local networks via mDNS (Multicast DNS) and DNS-SD (DNS Service Discovery).]]></descr>
 		<version>%%PKGVERSION%%</version>
 		<configurationfile>avahi.xml</configurationfile>
+		<plugins>
```

Comment:
This is not necessary, the `<plugins>` section is not read from this file.

Reply:
fixed